### DISCUSSION TITLE
Blade multiple subviews doesnt work

### DISCUSSION THREAD

**Discussion by clarkeash** [laravel/framework#33359] [General]
- Laravel Version: 7.17.2
- PHP Version: 7.4.5

### Description:
I am including 2 subviews in my blade view and I see the first one repeated.

### Steps To Reproduce:
I have the following views: [views.zip](https://github.com/laravel/framework/files/4840617/views.zip)
```
├── layouts
│   └── section.blade.php
├── sections
│   ├── one.blade.php
│   └── two.blade.php
└── welcome.blade.php
```

Sections `one` and `two` both extend the layout file, and `welcome` includes both `one` and `two`:

`welcome.blade.php`:
```
<!DOCTYPE html>
<html>
    <body>
        @include('sections.one')
        @include('sections.two')
    </body>
</html>
```

I just get the contents of `one` repeated:

<img width="612" alt="Screenshot 2020-06-27 at 15 42 39" src="https://user-images.githubusercontent.com/1612186/85923620-e4190a80-b88c-11ea-8fab-c67325431002.png">

I couldnt see a different way of doing this in the docs.

**Comment by rcbgalido** [comment]
Using the `@overwrite` directive seems to make it work.

`two.blade.php`:
```
@extends('layouts.section')

@section('title')
    Two
@overwrite

@section('description')
    Info about section 2
@overwrite
```
